public/css/main.css

```diff
@@ -339,41 +316,50 @@ strong {
 	flex: 1;
 }
 
-.row {
-	display: flex;
+/* register */
+.col {
+	width: 100%;
+	height: 75vh;
+	background: #EEEEEE;
+	margin: 1rem;
 }
 
-.col {
+.col > div:first-child {
+	background-color: #89d33c;
```

Comment:
That green is quite sharp. The G in the logo is a murkier #A6CA47.

Reply:
Yeah I thought so, the palleton said the green we've got works better though, will give the other one a try :D